src/subcommand/wallet/transaction_builder.rs

```diff
@@ -43,17 +43,18 @@ use {
 
 #[derive(Debug, PartialEq)]
 pub enum Error {
-  NotInWallet(SatPoint),
+  DuplicateAddress(Address),
+  Dust {
+    output_value: Amount,
+    dust_value: Amount,
+  },
   NotEnoughCardinalUtxos,
+  NotInWallet(SatPoint),
   UtxoContainsAdditionalInscription {
     outgoing_satpoint: SatPoint,
     inscribed_satpoint: SatPoint,
     inscription_id: InscriptionId,
   },
-  Dust {
-    output_value: Amount,
-    dust_value: Amount,
-  },
   ValueOverflow,
 }
 
```

Comment:
Maybe Target::Exact -> Target::TryExact ?

Reply:
Something to reflect that a non exact value can happen in some edge case like in this PR